data/style.css

```diff
@@ -5,4 +5,4 @@
ont-weight:700;line-height:51px;color:#006468;padding:0;margin-bottom:20px}.siimple-h1:not(:first-child){margin-top:28px}.siimple-h2{display:block;font-size:35px;font-weight:700;color:#17A1A5;padding:0;margin-bottom:20px}.siimple-h2:not(:first-child){margin-top:28px}.siimple-h3{display:block;font-size:25px;font-weight:700;color:#17A1A5;padding:0;margin-bottom:18px}.siimple-h3:not(:first-child){margin-top:28px}.siimple-h4{display:block;font-size:22px;font-weight:700;color:#17A1A5;padding:0;margin-bottom:16px}.siimple-h4:not(:first-child){margin-top:28px}.siimple-h5{display:block;font-size:20px;font-weight:700;color:#17A1A5;padding:0;margin-bottom:14px}.siimple-h5:not(:first-child){margin-top:28px}.siimple-h6{display:block;font-size:16px;font-weight:700;color:#17A1A5;padding:0;margin-bottom:12px}.siimple-h6:not(:first-child){margin-top:28px}.siimple-tag{display:inline-block;font-size:12px;text-decoration:none;line-height:22px;border-radius:5px;padding-left:6px;padding-right:6px;margin-right:1px}.siimple-tag.siimple-tag--navy{background-color:#17A1A5;color:#fff}.siimple-tag.siimple-tag--green{background-color:#1add9f;color:#fff}.siimple-tag.siimple-tag--teal{background-color:#18d2ba;color:#fff}.siimple-tag.siimple-tag--blue{background-color:#4894f0;color:#fff}.siimple-tag.siimple-tag--purple{background-color:#b490f5;color:#fff}.siimple-tag.siimple-tag--pink{background-color:#f45b93;color:#fff}.siimple-tag.siimple-tag--red{background-color:#ff1a4f;color:#fff}.siimple-tag.siimple-tag--orange{background-color:#ff8463;color:#fff}.siimple-tag.siimple-tag--yellow{background-color:#ffbf00;color:#fff}.siimple-tag.siimple-tag--grey{background-color:#ebf2fa;color:#17A1A5}.siimple-tag.siimple-tag--white{background-color:#fff;color:#17A1A5}.siimple-small{font-size:12px;color:#8790ab}.siimple-p,.siimple-paragraph{display:block;line-height:22px;font-size:14px;margin-top:0;margin-bottom:16px}.siimple-a,.siimple-link{color:#17A1A5;font-weight:700;text-decoration:underline;-webkit-transition:all .3s;-o-transition:all .3s;transition:all .3s}.siimple-a:hover,.siimple-link:hover{text-decoration:underline;cursor:pointer}.siimple-color--navy{color:#17A1A5}.siimple-color--green{color:#1add9f}.siimple-color--teal{color:#18d2ba}.siimple-color--blue{color:#4894f0}.siimple-color--purple{color:#b490f5}.siimple-color--pink{color:#f45b93}.siimple-color--red{color:#ff1a4f}.siimple-color--orange{color:#ff8463}.siimple-color--yellow{color:#ffbf00}.siimple-color--grey{color:#ebf2fa}.siimple-color--white{color:#fff}.siimple-bg--navy{background-color:#17A1A5}.siimple-bg--green{background-color:#1add9f}.siimple-bg--teal{background-color:#18d2ba}.siimple-bg--blue{background-color:#4894f0}.siimple-bg--purple{background-color:#b490f5}.siimple-bg--pink{background-color:#f45b93}.siimple-bg--red{background-color:#ff1a4f}.siimple-bg--orange{background-color:#ff8463}.siimple-bg--yellow{background-color:#ffbf00}.siimple-bg--grey{background-color:#ebf2fa}.siimple-bg--white{background-color:#fff}
```

Comment:
changing the attribute from 'display: inline-block' to 'display: inline' from 'label class siimply-label' fixes a display error of the checkbox and label of' save prefix and generate channel serial numbers'. These overlapped on small screens.